<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminFuntion</class>
 <widget class="QDialog" name="adminFuntion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>290</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="NewStadium">
    <attribute name="title">
     <string>New Stadium</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>241</width>
       <height>530</height>
      </rect>
     </property>
     <property name="title">
      <string>Add New Stadium</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stadium Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Team Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Street</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>City Info</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Capacity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Opened Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Grass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>AL / NL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NewStadium">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NewTeam">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Street">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CityInfo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PhoneNum">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Capacity">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Year">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="haveGrass">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Have Grass</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddNewStadium">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>470</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="noGrass">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>360</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>No Grass</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="AL">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>400</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>American League</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="NL">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>National League</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Month">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>January</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>February</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>March</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>April</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>May</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>June</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>July</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>August</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>September</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>October</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>November</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>December</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_Day">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>100</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Day</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Display_Stadium_List_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>570</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Display List</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="New_Team">
    <attribute name="title">
     <string>New Team</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Add New Team</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Team Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stadium Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_TeamSwitch">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_StadiumSwitch">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AddNewTeam">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Display_Stadium_List">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Display List</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Souvenir_List">
    <attribute name="title">
     <string>Souvenir List</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Add Souvenir</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Add_Souvenir">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_AddName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_AddPrice">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>241</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Delete Souvenir</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_deleteName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Delete_Souvenir">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>240</width>
       <height>170</height>
      </rect>
     </property>
     <property name="title">
      <string>Change Souvenir</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ChangeName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ChangePrice">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Change_Souvenir">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Display">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>560</y>
       <width>140</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string>Display List</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>511</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
